### DISCUSSION TITLE
GitHub benefits

### DISCUSSION THREAD

**Discussion by Antismart** [community/community#63634] [GitHub Education]
when do I get my benefits? I was approved on 9th August till now I haven't received them.

**Comment by AshishKumar9532** [comment]
same here,
Account is approved for education pack but no benefits yet.
Github please fix this issue.
![Screenshot (182)](https://github.com/community/community/assets/99255937/fed8a16a-154e-4e38-9173-9185e3f1f5e4)

**Comment by joy-ahmed** [comment]
If it's approved you can use benifit asap just visit the benefits page along with your credentials (make sure you logged in with approved account)

**Reply by Antismart** [reply]
Still can't get access to mine😔

**Reply by Antismart** [reply]
just received mine😊

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
